modules/ethereum/src/lib.rs

```diff
@@ -68,6 +74,16 @@ pub struct AuraConfiguration {
 	pub maximum_extra_data_size: u64,
 }
 
+/// Transaction pool configuration.
+///
+/// This is used to limit number of unsigned headers transactions in
+/// the pool. We never use it to verify signed transactions.
+pub struct PoolConfiguration {
+	/// Maximal difference between number of header from unsigned transaction
+	/// and current best block. This must be selected with caution - we can't
```

Comment:
We can't change it?

Reply:
It depends where config comes from - if it is hardcoded (like now), then it can be changed by changing runtime. We an put it to storage, to allow changing it without updating runtime. Could you, please, elaborate - why and when do you want to change it?

Reply:
I left the comment because the sentence seemed incomplete.